app/controllers/stories_controller.rb

```diff
@@ -236,6 +236,7 @@ def handle_user_index
     # 2nd with 1 badge (!) <-- and that would look off.
     @badges_limit = 6
     @profile = @user.profile.decorate
+    @flag_status = Reaction.where(user_id: session_current_user_id, reactable: @user).any?
```

Comment:
Just to double-check, is this safe to use with our caching?

Reply:
I _think_ so? I poked around and it seems like `session_current_user_id` is used in a few different controllers, and we have [this check](https://github.com/forem/forem/blob/master/app/controllers/concerns/edge_cache_safety_check.rb) in place 🤔

Reply:
Yes, that's the conclusion I had reached too, but when it comes to our caching I usually like to double or triple check.